examples/scroll/lv_example_scroll_7.c

```diff
@@ -63,6 +67,8 @@ static void update_scroll(lv_obj_t * obj)
     if(bottom_num != bottom_num_original) {
         lv_label_set_text_fmt(low_label, "current smallest\nloaded value:\n%"PRId32, bottom_num);
     }
+
+    lv_obj_set_user_data(obj, (void *)(uintptr_t)false);
```

Comment:
Idea: why not just store a NULL since this is what the expression expands to anyways and above store `obj` again in the user data? Reduces the amount of casting.

Reply:
I agree about the casting being distracting. `NULL` is a nice `false`, but I can't get on board with `obj` as the non-`NULL` without confusing readers. Check what I just pushed. This example is already using global variables so another one can be appropriate.

This example shows the working principle but I was wondering if users would benefit from something easier to copy-past into their projects. I.e. callbacks for loading and dropping items. See the question in #7388 after it was merged.